### PR TITLE
Yieldlab Bid Adapter: read and pass UserIdsAsEids atype information

### PR DESCRIPTION
**Pull request by nkloeber**
## Type of change

- [x] Feature

## Description of change

Read atype information from UserIdsAsEids and pass it as query parameter (atypes={idprovider}:{atype},{idprovider2}:{atype2},...)

## Other information

One approving review from a Yieldlab team member required. (@brushmate, @rey1128, @Mufas61 or @kippsterr)

### CODE REVIEW

File: modules/yieldlabBidAdapter.js [hunk 1]
```diff
@@ -303,6 +304,21 @@ function createUserIdString(eids) {
...
+ * @returns {String} idprovider:atype,idprovider2:atype2,...
+ */
+function createUserIdAtypesString(eids) {
+  const str = []
+  for (let i = 0; i < eids.length; i++) {
+    if (eids[i].uids[0].atype) {
+      str.push(eids[i].source + ':' + eids[i].uids[0].atype)
+    }
+  }
+  return str.join(',')
```
Comment (patmmccann) [inline modules/yieldlabBidAdapter.js:319]: Use semicolons please

Comment (kippsterr) [reply modules/yieldlabBidAdapter.js:319]: Hi @patmmccann,

we have not been using semicolons anywhere in the file so far. Also running `gulp lint` does not print any issues.

Would it make sense to add that kind of stuff to the linting rules and thus enforce consistency everywhere?

Comment (patmmccann) [reply modules/yieldlabBidAdapter.js:319]: we've attempted to do so, but fixing all the issues is a work in progress

https://github.com/prebid/Prebid.js/pull/8846

Comment (kippsterr) [reply modules/yieldlabBidAdapter.js:319]: That sounds reasonable. I have created #9386 to generally fix that for our adapter and will add semi colons to this PR as well.

**Review by ChrisHuie — APPROVED** [review]

**Pull request merged**